### PR TITLE
Add Withdrawal Admin View.

### PR DESCRIPTION
**Pull request by starkshaw**

### CODE REVIEW

**Comment by starkshaw** [comment]
Hold on, fix bugs on Withdrawal panel on user dashboard.

**Comment by starkshaw** [comment]
@danielpaul This branch is ready to be merged to UI.

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -62,5 +62,19 @@
         end
       end # column
     end # columns
+
+    columns do
+      column do
+        panel "Recent Withdrawal Requests" do
+          table_for Withdrawal.order('id desc').limit(10).each do |withdrawal|
```
Comment (danielpaul) [inline app/admin/dashboard.rb:69]: Could we just show the ones that are yet to be processed on the dashboard?

File: app/admin/withdrawal.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  filter :approved
+  filter :created_at
+  filter :updated_at
+
+  show :title => :id do
+    attributes_table do
+      row :id
+      row :shop_id
+      row :amount
+      row :approved
```
Comment (danielpaul) [inline app/admin/withdrawal.rb:27]: We would need an edit page for editing the records?

File: app/admin/dashboard.rb (hunk 1 above)
Comment (starkshaw) [inline app/admin/dashboard.rb:69]: What do you mean?

File: app/admin/withdrawal.rb (hunk 2 above)
Comment (starkshaw) [inline app/admin/withdrawal.rb:27]: To approve the withdraw request, duh.

Comment (danielpaul) [inline app/admin/withdrawal.rb:27]: I know but i'm saying that we need the form and the editable fields to update the record. You shouldn't be able to change the shop_id or amount. Just the status.

File: app/admin/dashboard.rb (hunk 1 above)
Comment (danielpaul) [inline app/admin/dashboard.rb:69]: Create a scope which will return to you just the records that are not approved yet. http://guides.rubyonrails.org/active_record_querying.html#scopes

Comment (danielpaul) [inline app/admin/dashboard.rb:69]: So you can do something like `Withdrawal.unapproved....`

File: app/admin/withdrawal.rb (hunk 2 above)
Comment (starkshaw) [inline app/admin/withdrawal.rb:27]: OK. Another issue: I don't know why the amount in database is stored with the unit of cent – € 5 will be 500 in it. I don't know how to fix this in terms of showing the correct value.

Comment (danielpaul) [inline app/admin/withdrawal.rb:27]: You could just divide by 100 to get the decimal value. You can create a new method in the model called `display_amount` or something like that and then in that just do the division. You can then use that method to show the amount.

**Comment by starkshaw** [comment]
@danielpaul Finished. Please review.

**Pull request merged**